Get_Repo_Details/config.py

```diff
@@ -0,0 +1,30 @@
+## Configuration constants
+import os
+
+BASE_URL = "https://api.github.com/graphql" # Github base url
+GITHUB_TOKEN = os.environ['GITHUB_TOKEN']  # Github token environment variable
+REPONAME, USERNAME = None, None
```

Comment:
this is supposed to be entered as a single string

Reply:
Okay I made this into a command line tool using fire. Is it okay?

Reply:
ya